R/io.R

```diff
@@ -258,7 +258,7 @@ write_civis.character <- function(x, tablename, database = NULL, if_exists = "fa
           credential_id = credential_id),
       import_args)
     job_r <- do.call(start_import_job, args)
-    put_r <- httr::PUT(job_r[["uploadUri"]], body = httr::upload_file(x))
+    put_r <- httr::RETRY("PUT", job_r[["uploadUri"]], body = httr::upload_file(x), terminate_on = c(403, 404))
```

Comment:
I think it'd be better to terminate on any 4xx error other 429 here (e.g., `terminate_on  = setdiff(400:499, 429)`).  That's what AWS recommends [here](https://docs.aws.amazon.com/general/latest/gr/api-retries.html), and this URI and the one for `write_disk` below will be S3 URLs.

Reply:
sure I can do that! `setdiff()` is a super clean way to express it, I like it 😀

Reply:
addressed the comments in https://github.com/civisanalytics/civis-r/pull/235/commits/7cdb800edc26da45911089a4bab13e3ee9709a0f